content/en/docs/examples/virtual-machines/multi-network/index.md

```diff
@@ -75,22 +67,31 @@ The following example updates the `/etc/hosts` file with the {{< gloss >}}Istiod
    /etc/hosts is an easy to use example. It is also possible to use a real DNS and certificate for Istiod, this is beyond
    the scope of this document.
 
+1. Obtain `tcp-istiod` port for Istio ingress gateway and export it as `CAPORT`. Add it into `/var/lib/istio/envoy/sidecar.env`.  Revisit the [preparing the cluster](#preparing-the-kubernetes-cluster-for-vms) section to learn how to obtain the `tcp-istiod` port.
+
+    {{< text bash >}}
+    $ echo "CA_ADDR=istiod.istio-system.svc:$CAPORT" >> cluster.env
+    $ echo "ISTIO_PILOT_PORT=$CAPORT >> cluster.env
+    {{< /text >}}
+
+    {{< tip >}}
+    Ensure you have used @.name=="tcp-istiod" to extract the `tcp-istiod` port for Istio ingress gateway
```

Comment:
Unclear what this means. What is "@.name"? Ensure you used it in which instructions step?

Reply:
thanks for catching this, I fixed the reference link.